mlir-tensorrt/compiler/lib/CAPI/Compiler/Compiler.cpp

```diff
@@ -99,6 +101,37 @@ MTRT_Status mtrtCompilerClientDestroy(MTRT_CompilerClient client) {
   return mtrtStatusGetOk();
 }
 
+//===----------------------------------------------------------------------===//
+// MTRT_OptionsContext
+//===----------------------------------------------------------------------===//
+
+MLIR_CAPI_EXPORTED MTRT_Status mtrtOptionsContextCreateFromArgs(
+    MTRT_CompilerClient client, MTRT_OptionsContext *options,
+    const char *optionsType, const MlirStringRef *argv, unsigned argc) {
+  std::vector<llvm::StringRef> argvStrRef(argc);
+  for (unsigned i = 0; i < argc; i++)
+    argvStrRef[i] = llvm::StringRef(argv[i].data, argv[i].length);
+
+  auto result = unwrap(client)->createOptions(optionsType, argvStrRef);
+  if (!result.isOk())
+    return wrap(result.getStatus());
+
+  *options = wrap(result->release());
+  return mtrtStatusGetOk();
+}
+
+MLIR_CAPI_EXPORTED MTRT_Status
+mtrtOptionsContextPrint(MTRT_OptionsContext options) {
+  unwrap(options)->print(llvm::outs());
+  return mtrtStatusGetOk();
+}
+
+MLIR_CAPI_EXPORTED MTRT_Status
```

Comment:
This can just return `void`.

Reply:
Sure, do you want me to change the other destroy functions as well?

Reply:
Try to keep your changes concentrated just to the scope of this PR

Reply:
I can change this in a follow-up PR if desired.